backend/src/main/java/com/google/sps/notifs/EmailNotifier.java

```diff
@@ -128,32 +67,47 @@ private static Message createMessageWithEmail(MimeMessage emailContent)
    * @return the MimeMessage to be used to send email
    * @throws MessagingException if there was a problem accessing the Store
    */
-  private MimeMessage createEmail(String subject, String bodyText) throws MessagingException {
+  private MimeMessage createEmail(String recipientUsername, String subject, String bodyText)
+      throws MessagingException {
 
     Properties props = new Properties();
     Session session = Session.getDefaultInstance(props, /* authenticator= */ null);
 
     MimeMessage email = new MimeMessage(session);
 
     email.setFrom(new InternetAddress(APPLICATION_EMAIL));
-    email.addRecipient(RecipientType.TO, new InternetAddress(recipientEmail));
+    email.addRecipient(RecipientType.TO, new InternetAddress(recipientUsername + "@google.com"));
     email.setSubject(subject);
     email.setText(bodyText);
     return email;
   }
 
   /** Function that access its api and using it sends an email */
-  //   TODO(#35): Create a dummy email for ad lib itself to send emails.
   //   TODO(#36): Replace body to send real link to user instead of generic.
-  public void notifyUser() throws MessagingException, IOException, GeneralSecurityException {
-    NetHttpTransport HTTP_TRANSPORT = GoogleNetHttpTransport.newTrustedTransport();
+  public void sendMatchEmail(String firstMatchRecipientName)
+      throws MessagingException, IOException {
+    MimeMessage email =
+        createEmail(
+            firstMatchRecipientName,
+            "Ad-lib: We found you a match !",
+            " We found you a match with matchUsername. "
+                + " Check your calendar for your meeting event,"
+                + " and feel free to join the Meet call now!\n");
+    Message messageWithEmail = createMessageWithEmail(email);
+    service.users().messages().send("me", messageWithEmail).execute();
+  }
+
+  /** Function that access its api and using it sends an email */
+  //   TODO(#36): Replace body to send real link to user instead of generic.
+  public void sendExpiredEmail(String expiredRecipientName) throws MessagingException, IOException {
     MimeMessage email =
         createEmail(
-            "Ad-Lib Meeting Found",
-            " Hey "
-                + recipientName
-                + " Please Join your Ad-Lib meeting via the link below : \n"
-                + " http://meet.google.com/new");
+            expiredRecipientName,
+            "Ad-lib: Sorry, we couldn't find you a match!",
+            " We couldn't find a match  for you this time, but we encourage you to please try "
```

Comment:
tiny nit: extra space between match and for

Reply:
Ack